api/src/apps/parkeergarages/tests/test_scraper.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+
+from apps.parkeergarages.models import GuidanceSignRaw, ParkingLocationRaw
+
+
+@patch('apps.parkeergarages.scraper.ParkingLocationScraper.fetch', autospec=True)
+class TestParkingLocationScraper(TestCase):
+    def test_ok(self, fetch):
+        fetch.side_effect = 'test'
```

Comment:
Make clear that 'test' is the returned json. E.g.:

`{'data': 'this-is-mock-json'}`